Holovibes/cuda_sources/convolution.cu

```diff
@@ -46,8 +46,8 @@ void convolution_kernel(float* gpu_input,
     // At this point, cuComplex_buffer is the FFT of the input
 
```

Comment:
You miss the `convolution.cuh` file. And the doc comment is not right for the function on the header.